src/Extensions.cs

```diff
@@ -113,6 +113,11 @@ public static string[] ToWords(this string value)
             return WordSplitPattern.Split(value).Where(s => !string.IsNullOrEmpty(s)).ToArray();
         }
 
+        public static string ToIdiomaticShortName(this string value)

```

Comment:
Shouldn't all names be Idiomatic? Maybe a more appropriate name would be `ToShortName`

Reply:
Fixed